table-cut-columns/table-cut-columns.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+"Cut selected columns from table. Assumes that the input file has a
+header line with column names"
+"""
+
+import sys
+import argparse
+import pandas as pd
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("-i", "--table",
+                    help="Input table file. Reads from STDIN if '-'")
+parser.add_argument("-c", "--col-name", nargs='+',
+                    help="Column name(s); selects column(s)")
+parser.add_argument("-o", "--col-name-as", nargs='+', required=False,
+                    help="Optional output column name. Equal times and same order as col-name")
+parser.add_argument("-d", "--sep", default="\t",
+                    help="Column separator character, (default: <TAB>)")
+
+args = parser.parse_args()
+
+def get_filehandle(filename):
+    """ Returns a filehandle """
+    if filename == "-":
+        return sys.stdin
+    return open(filename, "r")
+
+ifile = get_filehandle(args.table)
+
+df = pd.read_csv(ifile, sep=args.sep)
+
+if not (args.table):
+    print("Please specify the table name or path")
+    sys.exit(1)
+else:
```

Comment:
You can add `required = True` for `args.table` option and you don't have to do this check.